packages/context/src/context.ts

```diff
@@ -154,15 +304,16 @@ export class Context {
    *
    * @param keyWithPath The binding key, optionally suffixed with a path to the
    *   (deeply) nested property to retrieve.
+   * @param options Options for resolution
    * @returns A promise of the bound value.
    */
-  get(key: string, session?: ResolutionSession): Promise<BoundValue> {
+  get(keyWithPath: string, options?: ResolutionOptions): Promise<BoundValue> {
```

Comment:
This is a breaking change in our public API. The commit message should include a short description of how to upgrade existing code, see https://conventionalcommits.org/. That way the auto-generated changelog/release notes will include this important information.

(same comment applies to `ctx.getSync` and possible other places too)